zest/releaser/tests/fullrelease.txt

```diff
@@ -103,7 +105,7 @@ Now run the fullrelease:
     lists of files in version control and sdist match
     Changelog entries for version 0.2:
     <BLANKLINE>
-    0.2 (1972-12-25)
+    0.2 (unreleased)
```

Comment:
In this part the changelog is shown before asking for a version. This was added recently and not released yet. This has to do with reading the history twice, which since this pull request we no longer do.
In other words, this changes the merged pull request #156 slightly.